addons/web_editor/static/lib/odoo-editor/src/style.css

```diff
@@ -1,5 +1,5 @@
 .oe-floating {
-    box-shadow: 0px 3px 18px rgb(0 0 0 / 23%);
+    box-shadow: 0px 3px 18px rgba(0, 0, 0, .23);
```

Comment:
I would have kept the name odoo-editor.css. Also didn't you say you converted to scss ?

Reply:
I haven't found a preprocessor of scss client side and came to the conclusion that it is not important for this task so I've skipped it.

The name `odoo-editor.css` was in odoo when it was conceptually "bundled". I think it make more sense to call it `style.css` now.

Reply:
Fine by me for the scss. Out of scope indeed.
`style.css` doesn't say anything though, I would keep Odoo's conventions. Everywhere else you have descriptive file names, never just `style.css`.